Restructure Source Maps Upload page (#5799)
getsentry/sentry-docs: src/platform-includes/sourcemaps/primer/javascript.nextjs.mdx

> To show readable stack traces in your Sentry errors, once configured, the `@sentry/nextjs` SDK will generate and upload source maps automatically.

Reviewer: ```suggestion
`@sentry/nextjs` will generate and upload source maps automatically, in order to enable errors to have readable stacktraces.
```

Not wedded to this wording specifically, but as it stands the `once configured` naturally attaches itself to the first clause, not the second. I found myself having to go back and re-read the sentence, which is what I'm trying to avoid in my rewording.

> To upload source maps, the `@sentry/nextjs` SDK uses the [Sentry Webpack Plugin](https://github.com/getsentry/sentry-webpack-plugin) under the hood. See the [Manual Configuration](../manual-setup/) page and the Sentry [Webpack Plugin documentation](https://github.com/getsentry/sentry-webpack-plugin#cli-configuration) for more details.

Reviewer: Should we link them specifically to the sourcemaps section of the manual setup page?
